src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationusestyle/InputAnnotationUseStyleCompactNoArray.java

```diff
@@ -10,27 +10,29 @@
 package com.puppycrawl.tools.checkstyle.checks.annotation.annotationusestyle;
 
 @Deprecated
-@SomeArrays(pooches={DOGS.LEO}) // violation
-@SuppressWarnings({""}) // violation
+@SomeArrays(pooches={DOGS.LEO})// violation 'Annotation style must be 'COMPACT_NO_ARRAY''
+@SuppressWarnings({""}) // violation 'Annotation style must be 'COMPACT_NO_ARRAY''
 public class InputAnnotationUseStyleCompactNoArray
 {
 
 }
 
-@SomeArrays(pooches={DOGS.LEO}, um={}, duh={"bleh"}) // violation
+//violation below 'Annotation style must be 'COMPACT_NO_ARRAY''
+@SomeArrays(pooches={DOGS.LEO},um={}, duh={"bleh"})
 @SuppressWarnings("") //compact_no_array
 @Deprecated()
 class Dep6 {
 
 }
 
 @Deprecated
-@SomeArrays(pooches={DOGS.LEO}) // violation
-@SuppressWarnings({""}) // violation
+@SomeArrays(pooches={DOGS.LEO})// violation 'Annotation style must be 'COMPACT_NO_ARRAY''
```

Comment:
Restore spaces that are not required to be removed to comply to line length limit

Reply:
Done. Respective changes have been made. All spaces have been restored wherever applicable.